src/examples/merge-with-parent.yml

```diff
@@ -0,0 +1,23 @@
+description: >
+  Merge with a defined or detected parent branch to run tests against likely 
+  result.
+
+usage:
+  version: 2.1
+
+  orbs:
+    build-tools: circleci/build-tools@x.y.z
```

Comment:
Would recommend x.y (or the expected version number) rather than setting patch level.